BGAnimations/ScreenGameplay underlay/PerPlayer/nice.lua

```diff
@@ -13,23 +13,15 @@ if ThemePrefs.Get("nice") > 0 then
 			local pss = STATSMAN:GetCurStageStats():GetPlayerStageStats(player)
 			local PercentDP = pss:GetPercentDancePoints()
 			local percent = FormatPercentScore(PercentDP):gsub("%%", "")
-			-- pss:GetCurrentCombo() ignores potential "Miss combo"
-			-- so get the text from the Combo actor instead if it exists
 			local combo = nil
 
 			if not mods.HideCombo then
-				local player_actor = SCREENMAN:GetTopScreen():GetChild("Player"..pn)
-				if player_actor then
-					local combo_actor = player_actor:GetChild("Combo")
-					if combo_actor then
-						combo = combo_actor:GetChild("Number"):GetText()
-					end
-				end
+				combo = pss:GetCurrentCombo() + pss:GetCurrentMissCombo()
```

Comment:
This looks so strange lol. I guess the assumption is that one of these will always be 0. Would be good to add that as a comment.

Reply:
Oh, sorry for that, lol. I added a comment.